components/Header/index.stories.tsx

```diff
@@ -18,7 +19,7 @@ const meta: Meta = {
   ],
   parameters: {
     a11y: {
-      element: '#root :not([class*="mockBlock"]):not([class*="mockBlock"] *)',
+      element: `#root :not(.${style.mockBlock}):not(.${style.mockBlock} *)`,
```

Comment:
Ha, after a second thought. I took another approach, so it won't depend on css loader's naming configuration: 
```js
loader.options.modules = {
  auto: true,
  localIdentName: '[name]_[local]__[hash:base64:5]',
}
```
The default name is just a hash string, and `class*='mockBlock'` won't match.

Reply:
That config just adds a hash to the end. It looks like `mockBlock` (or whatever class name is in the css file) is the `local` variable in that config so `class*="mockBlock"` will work. However happy with your approach as that is going to use the full class.

![image](https://user-images.githubusercontent.com/15952529/116927583-f71c1880-acaf-11eb-84fd-e74db30c99d8.png)